src/DynamoCoreWpf/Views/Core/DynamoOpenFileDialog.cs

```diff
@@ -94,7 +94,7 @@ public DialogResult ShowDialog()
 
                 IShellItem dialogResult;
                 _dialog.GetResult(out dialogResult);
-                dialogResult.GetDisplayName(SIGDN.SIGDN_FILESYSPATH, out _fileName);
+                dialogResult.GetDisplayName(SIGDN.SIGDN_DESKTOPABSOLUTEEDITING, out _fileName);
```

Comment:
Is this because the SIGDN_FILESYSPATH is not shown for some cases?

Reply:
Yes, in cases where zip file was part of the path, it threw an error.